src/components/playlists/Playlist.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import PlaylistCard from "./PlaylistCard";
+
+const PlaylistContainer = () => {
+  const demoPlaylist = [
+    "Playlist 1",
+    "Playlist 2",
+    "Playlist 3",
+    "Playlist 4",
+    "Playlist 5",
+    "Playlist 6",
+    "Playlist 7",
+  ];
```

Comment:
Each playlist should contain the videos in that playlist

Reply:
I'll edit it as array of objects while adding functionality